StreamerBotPlugin/Services/HttpService.cs

```diff
@@ -25,13 +25,17 @@ namespace Loupedeck.StreamerBotPlugin.Services
     using System;
     using System.Net.Http;
     using System.Threading;
+    using System.Threading.Tasks;
 
     using Models.Receive;
 
     using Newtonsoft.Json;
 
     public class HttpService : IDisposable
     {
+        public EventHandler OnSuccess { get; set; } = delegate { };
+        public EventHandler OnFailure { get; set; } = delegate { };
```

Comment:
Small note: My personal preference / convention is to have EventHandler near bottom of the file